packages/itwin/tree-widget/src/components/trees/ModelsTree.tsx

```diff
@@ -24,8 +25,8 @@ interface TreeViewModelInfo {
 export const ModelsTreeComponent = (props: ModelTreeProps) => {
   const [is2dToggleActive, setIs2dToggleActive] = useState<boolean>(false);
   const [is3dToggleActive, setIs3dToggleActive] = useState<boolean>(false);
-  const [icon2dToggle, setIcon2dToggle] = useState<string>("icon-visibility");
-  const [icon3dToggle, setIcon3dToggle] = useState<string>("icon-visibility");
+  const [icon2dToggle, setIcon2dToggle] = useState(<SvgVisibilityShow />);
```

Comment:
this makes sense, but does this seem weird to you? storing a component in state?

Reply:
Let me take a look, not sure why this was necessary to be in state, can't imagine any caller setting those icons.

Reply:
Removed from state.